cpp/tests/copying/shift_tests.cpp

```diff
@@ -54,10 +54,18 @@ std::unique_ptr<cudf::scalar> make_scalar(
 }
 
 template <typename T>
-auto lowest = std::numeric_limits<T>::lowest();
+constexpr auto highest()
+{
+  if constexpr (cudf::is_chrono<T>()) return T::max();
+  return std::numeric_limits<T>::max();
+}
 
 template <typename T>
-auto highest = std::numeric_limits<T>::max();
+constexpr auto lowest()
+{
+  if constexpr (cudf::is_chrono<T>()) return T::min();
+  return std::numeric_limits<T>::lowest();
```

Comment:
The std behavior of min is unfortunate. Maybe worth adding a comment or pointing to the cppreference for why we need to use lowest rather than min for floats.

Reply:
@mythrocks No, that's exactly what I was referring to. The fact that we use `T::min` but `std::numeric_limits<T>::lowest` rather than `std::numeric_limits_<T>::min` will likely leave a lot of people scratching their heads. @codereport I was just suggesting documenting why the function called on line 66 has a different name than the one on line 67.